conda-forge.yml

```
@@ -2,3 +2,7 @@ provider: {linux_aarch64: default, linux_ppc64le: default}
 conda_forge_output_validation: true
 conda_build:
   pkg_format: '2'
+azure:
+  settings_win:
+    pool:
+      vmImage: vs2017-win2016
```

The next best thing would be to try the 2019 image. This is what Microsoft recommends in the blog linked above

```suggestion
      vmImage: windows-2019
```

This is also consistent with what the `3.8` builds are using

https://github.com/conda-forge/python-feedstock/blob/121811a2e0a879f1fe8eb86d7ec57ee59c0d6397/conda-forge.yml#L1-L4